pkg/dao/schema/environment.go

```diff
@@ -35,6 +36,10 @@ func (Environment) Fields() []ent.Field {
 			Comment("ID of the project to belong.").
 			NotEmpty().
 			Immutable(),
+		field.String("type").
+			Comment("Type of the environment.").
+			NotEmpty().
+			Immutable(),
```

Comment:
add default?

Reply:
i prefer to keep this as a `MUST` argument.